.github/workflows/labeler.yml

```diff
@@ -1,16 +1,14 @@
 name: "Pull Request Labeler"
 on:
-  pull_request:
-    types: [opened, synchronize]
-    paths:
-      - 'content/**'
-    branches:
-      - '**'
+- pull_request_target
```

Comment:
It's very cool from the human labeler point of view 😍 

Currently, this workflow is triggered by every PRs, and workflows triggered via 'pull_request_target' have write permission to this upstream repository. 

Therefore, IMHO, it would be a good for me to explicitly or restrictively handle this workflow. (i.e. only for labeling PRs related to localization). Later we can consider gradually expanding the scope of this workflow later.